tidings/utils.py

```diff
@@ -58,17 +20,32 @@ def collate(*iterables, **kwargs):
     descending order rather than ascending.
 
     """
-    # TODO: Liberate into the stdlib.
     key = kwargs.pop('key', lambda a: a)
     reverse = kwargs.pop('reverse', False)
-
     min_or_max = max if reverse else min
-    peekables = [peekable(it) for it in iterables]
```

Comment:
Actually, what error were you having? collate() passes Python 3 tests just fine in more-itertools. I wonder what's going on.